### PR TITLE
completed javascript-objects - Aaron He

### PR DESCRIPTION
**Pull request by AaronHe09**
image

### CODE REVIEW

File: javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+student.livesInIrvine = false;
+student.previousOccupation = 'Frac Operator';
+console.log('Value of livesInIrvine: ', student.livesInIrvine);
+console.log('Value of previousOccupation: ', student.previousOccupation);
+
+vehicle.color = 'Silver';
+vehicle.isConvertible = false;
+console.log('Value of color: ', vehicle.color);
+console.log('Value of isConvertible: ', vehicle.isConvertible);
```
Comment (shawnkost) [inline javascript-objects/main.js:27]: These should be using bracket notation rather than dot notation

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Aaron!

File: javascript-objects/main.js (hunk 1 above)
Comment (AaronHe09) [reply javascript-objects/main.js]: image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**